src/python/tests/core/bot/fuzzers/libFuzzer/engine_test.py

```diff
@@ -21,6 +21,7 @@
 import tempfile
 import unittest
 
+import metrics
```

Comment:
nit: move to the import group after this one, and do `from metrics import logs` instead of importing just `metrics`.